{"nodes":[{"pos":[609,703],"content":"The exception that is thrown when an attempt is made to access an unloaded application domain.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is thrown when an attempt is made to access an unloaded application domain.","pos":[0,94]}]},{"pos":[716,1674],"content":"In the .NET Framework version 2.0, an AppDomainUnloadedException that is not handled in user code has the following effect:      -   If a thread was started in managed code, it is terminated. The unhandled exception is not allowed to terminate the application.      -   If a task is executing on a <xref:System.Threading.ThreadPool> thread, it is terminated and the thread is returned to the thread pool. The unhandled exception is not allowed to terminate the application.      -   If a thread started in unmanaged code, such as the main application thread, it is terminated. The unhandled exception is allowed to proceed, and the operating system terminates the application.       AppDomainUnloadedException uses the HRESULT COR_E_APPDOMAINUNLOADED, which has the value 0x80131014.       For a list of initial property values for an instance of AppDomainUnloadedException, see the <xref:System.AppDomainUnloadedException.%23ctor%2A> constructors.","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"In the .NET Framework version 2.0, an AppDomainUnloadedException that is not handled in user code has the following effect:      -   If a thread was started in managed code, it is terminated. The unhandled exception is not allowed to terminate the application.      -   If a task is executing on a <xref:System.Threading.ThreadPool> thread, it is terminated and the thread is returned to the thread pool. The unhandled exception is not allowed to terminate the application.      -   If a thread started in unmanaged code, such as the main application thread, it is terminated. The unhandled exception is allowed to proceed, and the operating system terminates the application.       AppDomainUnloadedException uses the HRESULT COR_E_APPDOMAINUNLOADED, which has the value 0x80131014.       For a list of initial property values for an instance of AppDomainUnloadedException, see the <xref:System.AppDomainUnloadedException.%23ctor%2A> constructors.","pos":[0,948],"nodes":[{"content":"In the .NET Framework version 2.0, an AppDomainUnloadedException that is not handled in user code has the following effect:      -   If a thread was started in managed code, it is terminated.","pos":[0,191]},{"content":"The unhandled exception is not allowed to terminate the application.","pos":[192,260]},{"content":"-   If a task is executing on a &lt;xref:System.Threading.ThreadPool&gt; thread, it is terminated and the thread is returned to the thread pool.","pos":[266,404],"source":"      -   If a task is executing on a <xref:System.Threading.ThreadPool> thread, it is terminated and the thread is returned to the thread pool."},{"content":"The unhandled exception is not allowed to terminate the application.","pos":[405,473]},{"content":"-   If a thread started in unmanaged code, such as the main application thread, it is terminated.","pos":[479,576]},{"content":"The unhandled exception is allowed to proceed, and the operating system terminates the application.","pos":[577,676]},{"content":"AppDomainUnloadedException uses the HRESULT COR_E_APPDOMAINUNLOADED, which has the value 0x80131014.","pos":[683,783]},{"content":"For a list of initial property values for an instance of AppDomainUnloadedException, see the &lt;xref:System.AppDomainUnloadedException.%23ctor%2A&gt; constructors.","pos":[790,948],"source":"       For a list of initial property values for an instance of AppDomainUnloadedException, see the <xref:System.AppDomainUnloadedException.%23ctor%2A> constructors."}]}]},{"pos":[4068,4163],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AppDomainUnloadedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class."},{"pos":[4176,4796],"content":"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as \"The target application domain has been unloaded.\" This message takes into account the current system culture.       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as \"The target application domain has been unloaded.\"","pos":[0,210],"source":"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as \"The target application domain has been unloaded.\""},{"content":"This message takes into account the current system culture.","pos":[211,270]},{"content":"The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.","pos":[277,391],"source":"       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id=\"ph1\">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[397,611],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]},{"pos":[5406,5532],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AppDomainUnloadedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with a specified error message."},{"pos":[5545,6082],"content":"The content of the `message` parameter is intended to be understood by humans. The caller of this constructor is required to ensure that this string has been localized for the current system culture.       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The content of the <ph id=\"ph1\">`message`</ph> parameter is intended to be understood by humans.","pos":[0,78],"source":"The content of the `message` parameter is intended to be understood by humans."},{"content":"The caller of this constructor is required to ensure that this string has been localized for the current system culture.","pos":[79,199]},{"content":"The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.","pos":[206,320],"source":"       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id=\"ph1\">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|","pos":[326,530],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The error message string.|"}]},{"pos":[6238,6275],"content":"The message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message that describes the error.","pos":[0,37]}]},{"pos":[7046,7162],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AppDomainUnloadedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialized data.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with serialized data."},{"pos":[7174,7382],"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md).","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.","pos":[0,113]},{"content":"For more information, see <bpt id=\"p1\">[</bpt>XML and SOAP Serialization<ept id=\"p1\">](~/add/includes/ajax-current-ext-md.md)</ept>.","pos":[114,208],"source":" For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md)."}]},{"pos":[7662,7711],"content":"The object that holds the serialized object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]},{"pos":[7807,7866],"content":"The contextual information about the source or destination.","needQuote":true,"needEscape":true,"nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]},{"pos":[8459,8660],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AppDomainUnloadedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."},{"pos":[8673,9641],"content":"The content of the `message` parameter is intended to be understood by humans. The caller of this constructor is required to ensure that this string has been localized for the current system culture.       An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The content of the <ph id=\"ph1\">`message`</ph> parameter is intended to be understood by humans.","pos":[0,78],"source":"The content of the `message` parameter is intended to be understood by humans."},{"content":"The caller of this constructor is required to ensure that this string has been localized for the current system culture.","pos":[79,199]},{"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.","pos":[206,388],"source":"       An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property."},{"content":"The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a null reference if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.","pos":[389,643],"source":" The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor."},{"content":"The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.","pos":[650,764],"source":"       The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|","pos":[770,959],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The error message string.|"}]},{"pos":[9823,9860],"content":"The message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message that describes the error.","pos":[0,37]}]},{"pos":[9934,10190],"content":"The exception that is the cause of the current exception. If the `innerException` parameter is not a null reference, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]},{"content":"If the <ph id=\"ph1\">`innerException`</ph> parameter is not a null reference, the current exception is raised in a <bpt id=\"p1\">&lt;xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> block that handles the inner exception.","pos":[58,256],"source":" If the `innerException` parameter is not a null reference, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception."}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.AppDomainUnloadedException\n  id: AppDomainUnloadedException\n  children:\n  - System.AppDomainUnloadedException.#ctor\n  - System.AppDomainUnloadedException.#ctor(System.String)\n  - System.AppDomainUnloadedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.AppDomainUnloadedException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: AppDomainUnloadedException\n  nameWithType: AppDomainUnloadedException\n  fullName: System.AppDomainUnloadedException\n  type: Class\n  summary: The exception that is thrown when an attempt is made to access an unloaded application domain.\n  remarks: \"In the .NET Framework version 2.0, an AppDomainUnloadedException that is not handled in user code has the following effect:  \\n  \\n-   If a thread was started in managed code, it is terminated. The unhandled exception is not allowed to terminate the application.  \\n  \\n-   If a task is executing on a <xref:System.Threading.ThreadPool> thread, it is terminated and the thread is returned to the thread pool. The unhandled exception is not allowed to terminate the application.  \\n  \\n-   If a thread started in unmanaged code, such as the main application thread, it is terminated. The unhandled exception is allowed to proceed, and the operating system terminates the application.  \\n  \\n AppDomainUnloadedException uses the HRESULT COR_E_APPDOMAINUNLOADED, which has the value 0x80131014.  \\n  \\n For a list of initial property values for an instance of AppDomainUnloadedException, see the <xref:System.AppDomainUnloadedException.%23ctor%2A> constructors.\"\n  example:\n  - \"This section contains two code examples. The first example demonstrates the effects of an AppDomainUnloadedException on various threads, and the second shows elementary application domain unloading.  \\n  \\n Example 1  \\n  \\n The following code example defines a `TestClass` class that can be marshaled across application domain boundaries and an `Example` class containing a `static` (`Shared` in Visual Basic) `ThreadProc` method. The `ThreadProc` method creates an application domain, creates a `TestClass` object in the domain, and calls a method of `TestClass` that unloads the executing domain, causing an AppDomainUnloadedException.  \\n  \\n The `TestClass` method is executed without exception handling from a <xref:System.Threading.ThreadPool> thread and from an ordinary thread, demonstrating that the unhandled exception terminates the task or thread but not the application. It is then executed with and without exception handling from the main application thread, demonstrating that it terminates the application if not handled.  \\n  \\n [!code-vb[AppDomainUnloadedException#1](~/add/codesnippet/visualbasic/t-system.appdomainunload_1.vb)]\\n [!code-cs[AppDomainUnloadedException#1](~/add/codesnippet/csharp/t-system.appdomainunload_1.cs)]  \\n  \\n Example 2  \\n  \\n The following code example creates and unloads an application domain, and demonstrates that an AppDomainUnloadedException is thrown on a subsequent attempt to access the unloaded domain.  \\n  \\n [!code-cpp[ADUnload#1](~/add/codesnippet/cpp/t-system.appdomainunload_2.cpp)]\\n [!code-vb[ADUnload#1](~/add/codesnippet/visualbasic/t-system.appdomainunload_2.vb)]\\n [!code-cs[ADUnload#1](~/add/codesnippet/csharp/t-system.appdomainunload_2.cs)]\"\n  syntax:\n    content: >-\n      [System.Runtime.InteropServices.ComVisible(true)]\n\n      public class AppDomainUnloadedException : SystemException\n  inheritance:\n  - System.Exception\n  - System.SystemException\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.AppDomainUnloadedException.#ctor\n  id: '#ctor'\n  parent: System.AppDomainUnloadedException\n  langs:\n  - csharp\n  name: AppDomainUnloadedException()\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException()\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException()\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System\n  summary: Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class.\n  remarks: \"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as \\\"The target application domain has been unloaded.\\\" This message takes into account the current system culture.  \\n  \\n The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  syntax:\n    content: public AppDomainUnloadedException ();\n    parameters: []\n  overload: System.AppDomainUnloadedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AppDomainUnloadedException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.AppDomainUnloadedException\n  langs:\n  - csharp\n  name: AppDomainUnloadedException(String)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(String)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(String)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System\n  summary: Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with a specified error message.\n  remarks: \"The content of the `message` parameter is intended to be understood by humans. The caller of this constructor is required to ensure that this string has been localized for the current system culture.  \\n  \\n The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  \\n|<xref:System.Exception.Message%2A>|The error message string.|\"\n  syntax:\n    content: public AppDomainUnloadedException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the error.\n  overload: System.AppDomainUnloadedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AppDomainUnloadedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.AppDomainUnloadedException\n  langs:\n  - csharp\n  name: AppDomainUnloadedException(SerializationInfo,StreamingContext)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(SerializationInfo,StreamingContext)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System\n  summary: Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with serialized data.\n  remarks: This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md).\n  syntax:\n    content: protected AppDomainUnloadedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.AppDomainUnloadedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AppDomainUnloadedException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.AppDomainUnloadedException\n  langs:\n  - csharp\n  name: AppDomainUnloadedException(String,Exception)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(String,Exception)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(String,Exception)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System\n  summary: Initializes a new instance of the <xref href=\"System.AppDomainUnloadedException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.\n  remarks: \"The content of the `message` parameter is intended to be understood by humans. The caller of this constructor is required to ensure that this string has been localized for the current system culture.  \\n  \\n An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.  \\n  \\n The following table shows the initial property values for an instance of <xref:System.AppDomainUnloadedException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|The inner exception reference.|  \\n|<xref:System.Exception.Message%2A>|The error message string.|\"\n  syntax:\n    content: public AppDomainUnloadedException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the error.\n    - id: innerException\n      type: System.Exception\n      description: The exception that is the cause of the current exception. If the `innerException` parameter is not a null reference, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.\n  overload: System.AppDomainUnloadedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.SystemException\n  isExternal: false\n  name: System.SystemException\n- uid: System.AppDomainUnloadedException.#ctor\n  parent: System.AppDomainUnloadedException\n  isExternal: false\n  name: AppDomainUnloadedException()\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException()\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException()\n- uid: System.AppDomainUnloadedException.#ctor(System.String)\n  parent: System.AppDomainUnloadedException\n  isExternal: false\n  name: AppDomainUnloadedException(String)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(String)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.AppDomainUnloadedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.AppDomainUnloadedException\n  isExternal: false\n  name: AppDomainUnloadedException(SerializationInfo,StreamingContext)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(SerializationInfo,StreamingContext)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.AppDomainUnloadedException.#ctor(System.String,System.Exception)\n  parent: System.AppDomainUnloadedException\n  isExternal: false\n  name: AppDomainUnloadedException(String,Exception)\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException(String,Exception)\n  fullName: System.AppDomainUnloadedException.AppDomainUnloadedException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.AppDomainUnloadedException.#ctor*\n  parent: System.AppDomainUnloadedException\n  isExternal: false\n  name: AppDomainUnloadedException\n  nameWithType: AppDomainUnloadedException.AppDomainUnloadedException\n"}